admin_pages/messages/templates/ee_msg_details_main_add_meta_box.template.php

```diff
@@ -8,14 +9,17 @@
  * @var string     $action_message
  * @var string     $edit_message_template_form_url
  */
+
+use EventEspresso\core\services\request\sanitizers\AllowedTags;
+
 ?>
 
 <div id="admin-primary-mbox-dv" class="admin-primary-mbox-dv">
 
     <h4 class="admin-primary-mbox-h4">
-        <?php echo $event_name ? esc_html($event_name . ' Custom Template') : ''; ?>
+        <?php echo ($event_name ? esc_html($event_name . ' Custom Template') : ''); ?>
```

Comment:
I fixed the code but regarding the extra brackets for conditional prints, we already talked here: https://github.com/eventespresso/event-espresso-core/pull/3771#discussion_r831283816

If you believe that I should remove them just let me know.
Also, I'm using PHPstorm and I set the PSR12 for the EE project and reformatted the code. Please check it and let me know your thoughts.

Reply:
ohhhhh  i see... ok.. i was thinking that was ok for just those ternaries that were setting the values to `"true"` or `"false"`, but you wanted to do that for all ternaries used in templates.

hmmm....

ya ok that's fine... but JUST FOR TEMPLATES!!! 😆 

in this exact situation it won't matter because this needs to be translated and escaped differently anyways